doc/pdf-styling.yaml

```diff
@@ -3,11 +3,9 @@ author:
  - Jonathan M. Spring
  - Eric Hatleback
  - Allen D. Householder
- - Art Manion
- - Madison Oliver
- - Vijay Sarvapalli
- - Deana Shick
+ - Vijay S. Sarvepalli
  - Laurie Tyzenhaus
+ - Charles G. Yarbrough
```

Comment:
Chuck's name was on the published SSVC 2.0 PDF, but he wasn't in this `.yaml` file that specifies the draft PDF